test/assertions_test.rb

```diff
@@ -81,6 +81,20 @@ def test_assert_statsd_call
     end
 
     assert_no_assertion_triggered do
+      @test_case.assert_statsd_increment('counter', times: 2, tags: ['foo:1']) do
+        StatsD.increment('counter', tags: { foo: 1 })
+        StatsD.increment('counter', tags: { foo: 1 })
+      end
+    end
+
+    assert_assertion_triggered do
```

Comment:
The assertion on this line (90, the one I'm commenting on) is failing, but it should pass. And yea that's the case with two different values.